<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorConfigDialog</class>
 <widget class="QDialog" name="EditorConfigDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QGroupBox" name="saveGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Save Config:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="m_AutoSaveEnabledCheckBox">
        <property name="text">
         <string>Auto save enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_AutoSaveIntervalLayout">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="m_AutoSaveIntervalLayout_2" stretch="0,0,0,0,0">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save every</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_AutoSaveIntervalSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>minutes.</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sceneGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Scene Visualizer:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0" columnminimumwidth="10,0,0">
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="showObjectsBoundingBoxesCheckBox">
        <property name="text">
         <string>bounding boxes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="showObjectsCheckBox">
        <property name="text">
         <string>Show Objects</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="showTrajectoryCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Trajectory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="showObjectsLabelsCheckBox">
        <property name="text">
         <string>labels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sequences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="m_MultiSelectionCheckBox">
        <property name="text">
         <string>Multi Selection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_OkButton</sender>
   <signal>clicked()</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjectsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onShowObjectsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjectsBoundingBoxesCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onShowObjectsBoundingBoxesChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjectsLabelsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onShowObjectsLabelsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTrajectoryCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onShowTrajecoriesChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_AutoSaveEnabledCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onAutoSaveEnabledChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_AutoSaveIntervalSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onAutoSaveIntervalChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MultiSelectionCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EditorConfigDialog</receiver>
   <slot>onSequenceMultiSelectionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShowObjectsChanged(int)</slot>
  <slot>onShowObjectsBoundingBoxesChanged(int)</slot>
  <slot>onShowObjectsLabelsChanged(int)</slot>
  <slot>onShowTrajecoriesChanged(int)</slot>
  <slot>onTrajectoryPartsDataChanged()</slot>
  <slot>onAutoSaveEnabledChanged()</slot>
  <slot>onAutoSaveIntervalChanged(int)</slot>
  <slot>onShowComplexObjectsChanged(int)</slot>
  <slot>onSequenceMultiSelectionChanged(int)</slot>
 </slots>
</ui>
